# define the goal
string[] grippers_names
geometry_msgs/Pose[] gripper_poses 
---
# define the result
smmap_msgs/SimulatorFeedback sim_state
